pkg/ebpf/netlink/conntracker.go

```diff
@@ -311,6 +339,10 @@ func (ctr *realConntracker) unregister(c ct.Con) int {
 	then := time.Now().UnixNano()
 	atomic.AddInt64(&ctr.stats.unregisters, 1)
 	atomic.AddInt64(&ctr.stats.unregistersTotalTime, then-now)
+	if misses > 0 {
+		log.Debugf("missed register event for: %s", conDebug(c))
```

Comment:
I think this should be `Tracef` as well

Reply:
I don't have a strong opinion about it, but I am expecting these entries now to be very rare and I think it would be useful to have them at debug level.